distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+https.protocols=TLSv1.1,TLSv1.2
+----
+
+[NOTE]
+====
+Depending on the environment, it may be easier for integrators and administrators to configure ${branding} using the ${admin-console} prior to disabling it for hardening purposes.
+The ${admin-console} can be re-enabled for additional configuration changes.
+====
+
+In an environment hardened for security purposes, access to the ${ddf-branding} ${admin-console} might be denied.
+It is necessary to configure ${branding} (e.g., providers, Schematron rulesets, etc.) using `.config` files or the ${admin-console}.
+Configuration via the Karaf command line console is not supported and may result in configuration errors.
+The OSGi container detects the creation of `.config` files in the `etc/` directory.
+The following sections describe how to configure each ${branding} item using both of these mechanisms.
+A template file is provided for some configurable ${branding} items so that they can be copied/renamed then modified with the appropriate settings.
+
+[WARNING]
+====
+If at a later time the ${admin-console} is enabled again, all of the configuration done via `.config` files is loaded and displayed.
+However, note that the name of the `.config` file is not used in the ${admin-console}.
+Rather, OSGi assigns a universally unique identifier (UUID) when the ${branding} item was created and displays this UUID in the console (e.g., `OpenSearchSource.112f298e-26a5-4094-befc-79728f216b9b`)
+====
+
+.Templates included with ${branding}
+[cols="1,4,4,1" options="header"]
+|===
+
+|${ddf-branding} Service
+|Template File Name
+|Factory PID
+|Configurable Properties
+
+|${ddf-branding} Catalog Framework
+|`ddf.catalog.impl.service.CatalogFrameworkImpl.cfg`
+|`ddf.catalog.CatalogFrameworkImpl`
+|Standard Catalog Framework
+
+|===
+
+==== Configuring Using a `.cfg` File Template
+
+The following steps define the procedure for configuring a new source or feature using a `config` file:
+
+. Copy/rename the provided template file in the ``etc/templates` directory to the `etc` directory. (Refer to the table above to determine correct template.)
+.. *Mandatory*: The dash between the PID (e.g., `OpenSearchSource_site.cfg`) and the instance name (e.g., `OpenSearchSource_site.cfg`) is required. The dash is a reserved character used by OSGi that identifies instances of a managed service factory that should be created.
+.. Not required, but a good practice is to change the instance name (e.g., `federated_source`) of the file to something identifiable (`source1- ddf`).
+. Edit the copied file to etc with the settings for the configuration. (Refer to the table above to determine the configurable properties).
+.. This file is a Java properties file, hence the syntax is `<key>` = `<value>`.
+.. Consult the inline comments in the file for guidance on what to modify.
+
+The new service can now be used as if it was created using the ${admin-console}.
+
+==== Configuring Notifications
```

Comment:
This section doesn't belong in the Managing DDF document. It should be moved to the Integrating document instead.